rails.md

```diff
@@ -0,0 +1,63 @@
+# Ruby on Rails
+
+An open source full-stack web application framework for the Ruby programming language.
+
+## Beginning Rails
+
+* Complete thoughtbot's [Intro to Ruby on Rails Tutorial](http://upcase.com/intro-to-ruby-on-rails)
+* Complete Michael Hartl's [Rails Tutorial](http://ruby.railstutorial.org)
+* Complete [Rails for Zombies](http://railsforzombies.org)
+* Push your code to [GitHub](http://github.com)
+* Deploy to [Heroku](http://heroku.com)
+
+### You should be able to
+
+* Create a route, controller, and action.
+* Query the database.
+* Create a form that saves records.
+* Associate ActiveRecord objects.
+* Validate ActiveRecord objects.
+* Whitelist ActiveRecord attributes.
+* Interpolate data in a view.
+* Deploy your app to production.
+
+## Intermediate Rails
+
+* Complete thoughtbot's [Intermediate Ruby on Rails Tutorial](http://upcase.com/intermediate-ruby-on-rails)
+* Read [Agile Web Development with Rails](http://pragprog.com/book/rails4/agile-web-development-with-rails-4)
+* Watch [Humans Present: Refactoring](https://upcase.com/videos/refactoring)
+* Complete the [RailsApps tutorials](https://tutorials.railsapps.org)
+* Watch [Railscasts](http://railscasts.com)
```

Comment:
I'd more readily recommend Destroy All Software - I feel like it holds up better.